### DISCUSSION TITLE
Новые свойства в Api

### DISCUSSION THREAD

**Discussion by KukushkaSex** [Sokol-Games-GmbH/PixelCombats.PublicApi#93] [Ideas]
Добавить свойство, возможные только для прочтения, например внутрь класса Properties, свойства:
`int skinid { get; }`
`float jump { get; } // значение прыжка `
`float speed { get; } // значение скорости `
Дальше описывать свойства не буду, думая вы поймёте сами.

**Comment by kkohno** [comment]
данные вещи можно на 2 группы разделить
1 - изменяемые , тоесть скин, его еще можно прочитать
2 - значения, типо длина прыжка не изменяемые и их не имеет смыла читать в апи... 

выпишите интересующие свойства группы 1

**Comment by KukushkaSex** [comment]
Имеет, можно будет уже свой античит делать прямо в режиме

**Comment by Kukushechka7-ezx** [comment]
Написал новый Issua об этом
